backends/advanced/src/advanced_omi_backend/config.py

```
@@ -165,4 +170,239 @@ def get_audio_storage_settings():
+
+    # Common locations (container with config dir mount vs repo root)
+    candidates = [Path("/app/config/config.yml"), Path("config.yml")]
+
+    # Also walk up from current file's parents defensively
+    try:
+        for parent in Path(__file__).resolve().parents:
+            c = parent / "config.yml"
+            if c.exists():
+                return c
+    except Exception:
+        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)